### ISSUE TITLE
bacula-fd launchd configuration

### ISSUE THREAD

**Issue by sts** [Homebrew/legacy-homebrew#45004]
Hello,

the following is a working launchd configuration, tested on OSX 10.10 with Bacula 5 & 7, which may be installed to ```/Library/LaunchDaemons/homebrew.mxcl.bacula-fd.plist```.

```
<?xml version="0.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <key>Label</key>
    <string>homebrew.mxcl.bacula-fd</string>
    <key>ProgramArguments</key>
    <array>
      <string>/usr/local/bin/bacula-fd</string>
      <!-- make sure this doesn't contain any spaces -->
      <string>-c/usr/local/etc/bacula-fd.conf</string>
      <string>-f</string>
    </array>
    <key>RunAtLoad</key>
    <true/>
    <key>UserName</key>
    <string>root</string>
  </dict>
</plist>
```

I also wanted to have mentioned previous bacula contributors for comments and corrections @jasperla @tgvaughan @ewindisch

**Comment by mikemcquaid** [comment]
Could you try and open a pull request? This should help and I'm happy to walk you through anything else: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged

Thanks!

**Comment by tdsmith** [comment]
Closed by #45121; thanks again!
